communitys_franken.json

```diff
@@ -251,7 +271,10 @@
 			"nodeMaps" : [{
 				"url" : "https://monitoring.freifunk-franken.de/map?mapcenter=49.13304,10.07652,14",
 				"mapType" : "geographical"
-			}]
+			}],
+			"socialprojects": {
+				"number": 2
+			},
```

Comment:
IIRC the script has no external dependencies, everything should be covered in Python 3.
Besides validating the JSON format can be done with any other JSON parser/validator.